### ISSUE TITLE
Standard GitHub Action for invoking OFT from within workflows

### ISSUE THREAD

**Issue by sophokles73** [itsallcode/openfasttrace#430]
It seems reasonable to run OFT trace on a code base as part of CI. However, doing this currently requires manual steps to install Java, assemble a command line and run the CliStarter class.

I would be nice to have a standard GitHub Action that does all that for you and that allows to provide the file patterns and e.g. the report format as inputs, so that it can be used in arbitrary environments.

**Comment by sophokles73** [comment]
I can imagine adding the (composite) Action to the existing OFT repository and tie it to OFT's release cycle/cadence. Alternatively, a separate repository would allow for more independence and also make it easier to pull in additional plugins (like the AsciiDoc plugin) without creating a dependency from the OFT repo to the plugin repo(s).

@kaklakariada @redcatbear WDYT?

**Comment by kaklakariada** [comment]
Having a GitHub action is a really good idea! I would prefer having a separate repository. It will be more effort, but more flexible.

Do you have experience with creating actions?

**Comment by sophokles73** [comment]
Yes, I am currently working on an OFT Action in the context of Eclipse uProtocol. However, I'd rather contribute it to OpenFastTrace directly ... If you set up a repository I can create a PR.

**Comment by kaklakariada** [comment]
@sophokles73 Sounds good!
I created https://github.com/itsallcode/openfasttrace-github-action and invited you as maintainer.

**Comment by redcatbear** [comment]
@kaklakariada, you beat me to it. I just wanted to do that. Sorry that you had to do that on your vacation.

**Comment by sophokles73** [comment]
Closed via https://github.com/itsallcode/openfasttrace-github-action/pull/1
